model/riscv_insts_vext_arith.sail

```diff
@@ -0,0 +1,531 @@
+                              let v_double : bits('m * 2) = EXTS(vs2_val[i]);
+                              slice(v_double >> shift_amount, 0, SEW) + EXTZ('m, rounding_incr)
+                            },
+        VV_VMINU         => to_bits(SEW, min(unsigned(vs2_val[i]), unsigned(vs1_val[i]))),
+        VV_VMIN          => to_bits(SEW, min(signed(vs2_val[i]), signed(vs1_val[i]))),
+        VV_VMAXU         => to_bits(SEW, max(unsigned(vs2_val[i]), unsigned(vs1_val[i]))),
+        VV_VMAX          => to_bits(SEW, max(signed(vs2_val[i]), signed(vs1_val[i]))),
+        VV_VRGATHER      => {
+                              assert(vs1 != vd & vs2 != vd);
+                              let idx = unsigned(vs1_val[i]);
+                              let VLMAX = int_power(2, LMUL_pow + VLEN_pow - SEW_pow);
+                              assert(VLMAX <= 'n);
+                              if idx < VLMAX then vs2_val[idx] else zeros() 
+                            },
+        VV_VRGATHEREI16  => {
+                              assert(vs1 != vd & vs2 != vd);
+                              /* vrgatherei16.vv uses SEW/LMUL for the data in vs2 but EEW=16 and EMUL = (16/SEW)*LMUL for the indices in vs1 */
+                              let vs1_new : vector('n, dec, bits(16)) = read_vreg(num_elem, 16, 4 + LMUL_pow - SEW_pow, vs1);
+                              let idx = unsigned(vs1_new[i]);
+                              let VLMAX = int_power(2, LMUL_pow + VLEN_pow - SEW_pow);
+                              assert(VLMAX <= 'n);
+                              if idx < VLMAX then vs2_val[idx] else zeros()                   
+                            }
+      }
+    }
+  };
+
+  write_vreg(num_elem, SEW, LMUL_pow, vd, result);
+  vstart = EXTZ(0b0);
+  RETIRE_SUCCESS
+}
+
+mapping vvtype_mnemonic : vvfunct6 <-> string = {
+  VV_VADD          <-> "vadd.vv",
+  VV_VSUB          <-> "vsub.vv",
+  VV_VAND          <-> "vand.vv",
+  VV_VOR           <-> "vor.vv",
+  VV_VXOR          <-> "vxor.vv",
+  VV_VRGATHER      <-> "vrgather.vv",
+  VV_VRGATHEREI16  <-> "vrgatherei16.vv",
+  VV_VSADDU        <-> "vsaddu.vv",
+  VV_VSADD         <-> "vsadd.vv",
+  VV_VSSUBU        <-> "vssubu.vv",
+  VV_VSSUB         <-> "vssub.vv",
+  VV_VSLL          <-> "vsll.vv",
+  VV_VSMUL         <-> "vsmul.vv",
+  VV_VSRL          <-> "vsrl.vv",
+  VV_VSRA          <-> "vsra.vv",
+  VV_VSSRL         <-> "vssrl.vv",
+  VV_VSSRA         <-> "vssra.vv",
+  VV_VMINU         <-> "vminu.vv",
+  VV_VMIN          <-> "vmin.vv",
+  VV_VMAXU         <-> "vmaxu.vv",
+  VV_VMAX          <-> "vmax.vv"
+}
+
+mapping clause assembly = VVTYPE(funct6, vm, vs2, vs1, vd)
+  <-> vvtype_mnemonic(funct6) ^ spc() ^ vreg_name(vd) ^ sep() ^ vreg_name(vs2) ^ sep() ^ vreg_name(vs1) ^ maybe_vmask(vm)
+
+
+/* *******************************OPIVX(Vector Slide & Gather Instructions)******************************** */
```

Comment:
If you're going to pad with *'s, do so to a fixed width, not some arbitrary number

Reply:
The format of headings are all modified.